pkg/projects/resource_project_user_test.go

```diff
@@ -0,0 +1,171 @@
+package projects
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+type ArtifactoryUser struct {
+	Email    string `json:"email"`
+	Password string `json:"password"`
+	Admin    bool   `json:"admin"`
+}
+
+func createTestUser(t *testing.T, projectKey string, username string, email string) {
+	restyClient := getTestResty(t)
+
+	user := ArtifactoryUser{
+		Email:    email,
+		Password: "Password1",
+		Admin:    false,
+	}
+
+	_, err := restyClient.R().SetBody(user).Put(fmt.Sprintf("/artifactory/api/security/users/%s", username))
```

Comment:
If the `id` is on the end, you don't need `Sprintf` at all, you can just use string concat `+`